test/cql-pytest/test_cast_data.py

```diff
@@ -119,6 +119,9 @@ def test_ignore_cast_to_the_same_type(cql, table1):
     assert cql.execute(f"SELECT CAST(p as int) FROM {table1} WHERE p={p}").one()._fields[0] == "p"
 
 # Test casting a counter to various other types. Reproduces #14501.
+@pytest.mark.parametrize("test_keyspace",
+                         [pytest.param("tablets", marks=[pytest.mark.xfail(reason="issue #19449")]), "vnodes"],
```

Comment:
I think this mentions the wrong issue - should be https://github.com/scylladb/scylladb/issues/18180
(the issue you mention is about disabling counters, the issue I want to list is the issue of fixing the counters).

Reply:
Fixed.